src/components/summary/examples/summary-grouped-validation/index.njk

```diff
@@ -29,7 +29,7 @@
                             },
                             {
                                 "rowTitle": "What was the value of the business's total sales of food?",
-                                "errorMessage": "Change one or more of the figures so they total the sum of £600",
+                                "errorMessage": "Change one or more of the figures so they sum to £600",
```

Comment:
Text needs changing, not sure what its meant to say

Reply:
@LauraC119?

Reply:
should be "sum to £600". This has made me notice, though, that while the visual example is a sum of pounds the content on the page is a percentage. @jrbarnes9 do we normally match the example to the page content?

Reply:
Yeah usually.  I'll update this to match.